modules/core/package.json

```diff
@@ -47,7 +47,7 @@
     "@bitgo/account-lib": "^2.3.0",
     "@bitgo/statics": "^5.3.0",
     "@bitgo/unspents": "^0.6.0",
-    "@bitgo/utxo-lib": "^1.7.1",
+    "@bitgo/utxo-lib": "git+https://github.com/BitGo/bitgo-utxo-lib.git#a08adf4478d8bace8a0e932158674e76ae463fca",
```

Comment:
Should we wait for utxo-lib to get a release?

Reply:
it is actually: https://github.com/BitGo/bitgo-utxo-lib/tree/v1.7.x but we decided to require it by hash for security reasons rather than a tag name.

Reply:
Could we release bitgo-utxo-lib rather than refer to a hash? This precludes dep hoisting with other semver-compatible utxo-lib consumers.

Reply:
@argjv could you weigh in on this? We kinda went back and forth...

Reply:
Oh, here we are using the new repo name. I just published 1.7.3 for `@bitgo/utxo-lib`.
I thought we were referencing the `bitgo-utxo-lib` package.

Reply:
fixed in 1798510690766438e4faa30bbb0c3f4188d99e91